src/domain/ruleset-repository.ts

```diff
@@ -218,7 +218,8 @@ function loadConfiguration(rulesetRepo: RulesetRepository): Configuration {
 
   let config: Record<string, any>;
   try {
-    config = yaml.parse(fs.readFileSync(rulesetRepo.configFilePath, "utf-8"));
+    config =
+      yaml.parse(fs.readFileSync(rulesetRepo.configFilePath, "utf-8")) || {};
```

Comment:
Here `yaml.parse` returns null for an empty file. The mistake was that I thought it returned `{}`.